docs/source/models/intervention_models/mncnh_pregnancy/iv_iron_antenatal/iv_iron_mncnh.rst

```diff
@@ -89,28 +88,64 @@ Hemoglobin exposure
 
 .. todo::
 
-  Update IV iron effect size to be consistent with new data from Chris T.
-  Also, assume no individual level heterogeneity despite having some data on this. (We chose not to model this in order to simplify the data prep for this model)
+  - Decide how we want to model uncertainty in the effect size of IV iron on hemoglobin concentrations.
+  - Assume no individual level heterogeneity despite having some data on this. (We chose not to model this in order to simplify the data prep for this model)
```

Comment:
I think the first TODO has been addressed, and the second TODO has become a limitation?